gcloudsdk/gcloudsdk.nuspec

```
@@ -6,20 +6,24 @@
   <metadata>
     <id>gcloudsdk</id>
     <!-- Use the signing time of the executable file as the version. -->
-    <version>0.0.0.20200527</version>
+    <version>0.0.0.20201022</version>
     <packageSourceUrl>https://github.com/gpoul/chocolatey-packages</packageSourceUrl>
     <owners>gpoul</owners>
     <title>Google Cloud SDK (Install)</title>
     <authors>Google</authors>
     <projectUrl>https://cloud.google.com/sdk/</projectUrl>
     <docsUrl>https://cloud.google.com/sdk/docs</docsUrl>
-    <tags>google cloud cli gcloud sdk admin</tags>
+    <tags>google cloud cli gcloud sdk admin gcloudsdk gcp</tags>
     <summary>SDK for Google Cloud</summary>
     <description>
 The Google Cloud SDK is a set of tools for Google Cloud Platform. It contains gcloud, gsutil, and bq, which you can use to access Google Compute Engine, Google Cloud Storage, Google BigQuery, and other products and services from the command-line. You can run these tools interactively or in your automated scripts.
 
 This package always installs the latest version of the Google Cloud SDK from Google. It first retrieves the most current installer, which then fetches the latest version of the SDK and installs it for all users in the default installation path. Updates can be performed through the gcloud cli.
     </description>
+     <dependencies>
++      <dependency id="python3" />
++      <dependency id="chocolatey-core.extension" version="1.1.0" />
++    </dependencies>
```

Here are some redundant '+' signs.
```suggestion
       <dependency id="python3" />
       <dependency id="chocolatey-core.extension" version="1.1.0" />
     </dependencies>
```